frontend/src/Admin/components/ShowSubmissionDetails.js

```diff
@@ -0,0 +1,158 @@
+    case PHOTO:
+      return (
+        <Fragment>
+          <PhotoThumbnail
+            alt={submission.title}
+            src={`//localhost:3000/static/uploads/${getImageThumbnail(
+              submission.path
+            )}`}
+          />
+          <dt>Dimensions</dt>
+          <dd>{submission.horizDimInch} in. × {submission.vertDimInch} in.</dd>
+          <dt>Media Type</dt>
+          <dd>{submission.mediaType}</dd>
+        </Fragment>
+      )
+    case VIDEO:
+      if (submission.provider === 'youtube') {
+        return (
+          <Fragment>
+            <dt>YouTube Video</dt>
+            <dd>
+              <a href={`${YOUTUBE_BASE_URL}${submission.videoId}`}>
+                {`${YOUTUBE_BASE_URL}${submission.videoId}`}
+              </a>
+            </dd>
+          </Fragment>
+        )
+      } else {
+        return (
+          <Fragment>
+            <dt>Vimeo Video</dt>
+            <dd>
+              <a href={`${VIMEO_BASE_URL}${submission.videoId}`}>
+                {`${VIMEO_BASE_URL}${submission.videoId}`}
+              </a>
+            </dd>
+          </Fragment>
+        )
+      }
+    case OTHER:
+      return (
+        <Fragment>
+          <dt>Other</dt>
+          <dd>
+            <a href={submission.path}>
+              {submission.path}
+            </a>
+          </dd>
+        </Fragment>
+      )
+    default:
+      console.error(`Unexpected Type ${submission.entryType}`, submission)
+      return null
+  }
+}
+
+const ShowSubmissionDetails = ({ submission }) => (
+  <Row>
+    <Col>
+      <h4 className='text-center'>Artist{submission.group ? 's' : null}</h4>
+      <dl>
+        <dt>Name</dt>
+        <dd>{submission.student.firstName} {submission.student.lastName} ({submission.student.username})</dd>
+        {submission.student.displayName ? (
+          <Fragment>
+            <dt>Artist Name</dt>
+            <dd>{submission.student.displayName}</dd>
+          </Fragment>
+        ) : null}
+        <dt>Year Level</dt>
+        <dd>{submission.yearLevel}</dd>
+        <dt>Academic Program</dt>
+        <dd>{submission.academicProgram}</dd>
+        {submission.group ? (
+          <Fragment>
+            <dt>Group Members</dt>
+            <dd>{submission.group.participants}</dd>
+          </Fragment>
+        ) : null}
+      </dl>
+    </Col>
+    <Col>
+      <h4 className='text-center'>Submission</h4>
+      <dl>
+        <dt>Title</dt>
+        <dd>{submission.title}</dd>
+        {submission.comment ? (
+          <Fragment>
+            <dt>Artist&quot;s Comment</dt>
+            <dd>{submission.comment}</dd>
+          </Fragment>
+        ) : null}
+        {renderSubmissionByType(submission)}
+        <dt>For Sale?</dt>
+        <dd>{submission.forSale ? 'Yes' : 'No'}</dd>
+        <dt>More Copies?</dt>
+        <dd>{submission.moreCopies ? 'Yes' : 'No'}</dd>
+      </dl>
+    </Col>
+  </Row>
+)
+
+ShowSubmissionDetails.propTypes = {
+  submission: PropTypes.shape({
```

Comment:
Perhaps interesting to include here:
* score
* invited status
* date submitted (if available)

Reply:
I had similar thoughts but decided to leave them out because I was rushing to get the PR in before our meeting. I'll find a creative way of including them

Reply:
would "date submitted" be the createdAt value?